CapstoneProject/Assets/Scripts/Core/Components/Characters/Playable_Character/PlayerAbilityComponent.cs

```diff
@@ -94,6 +102,69 @@ public GameObject ScanForLowestHealthEnemy()
         
         return lowestHealthTarget;
     }
+
+    public List<PlayerAbility> GetAbilities() => playerAbilities;
+
+    public void ChangeAbilties(int abilityone, int abilitytwo)
+    {
+        List<PlayerAbility> ability = new List<PlayerAbility>();
+        switch (abilityone)
+        {
+            case 1:
+                ability[0] = DashAbility;
+                break;
+            case 2:
+                ability[0] = ShieldAbility;
+                break;
+            case 3:
+                ability[0] = BowShootingAbility;
+                break;
+            case 4:
+                ability[0] = ProjectileShootingAbility;
+                break;
+            case 5 :
+                ability[0] = LightningStrikeAbility;
+                break;
+            case 0:
+               ability[0] = null;
+                break;
+            default:
+                Debug.Log(ability[0]);
+                Debug.Log("Ability one not found");
+                ability[0] = null;
+                break;
+        }
+        switch (abilitytwo)
```

Comment:
should we just make a function for this switch to save from it being duplicated? takes in the ability as one param and the ability to set as the other?